recipes/eglibc/eglibc.inc

```diff
@@ -24,14 +24,17 @@ LIBTOOL_DEPENDS = ""
 
 MACHINE[unexport]       = "1"
 
-RECIPE_FLAGS:native = "toolchain_kernel_version_native"
-RECIPE_FLAGS:machine = "toolchain_kernel_version_machine"
+RECIPE_FLAGS:native = "toolchain_kernel_version_native eglibc_obsolete_rpc"
+RECIPE_FLAGS:machine = "toolchain_kernel_version_machine eglibc_obsolete_rpc"
 RECIPE_FLAGS:sdk = "toolchain_kernel_version_sdk"
 
 DEFAULT_USE_toolchain_kernel_version_native	= "2.6.32"
 DEFAULT_USE_toolchain_kernel_version_machine	= "2.6.32"
 DEFAULT_USE_toolchain_kernel_version_sdk	= "2.6.32"
 
```

Comment:
Ok, I think we should consider enabling it by default on the master branch (until we have libtirpc in place).